package.json

```diff
@@ -21,8 +19,20 @@
   },
   "homepage": "https://github.com/francisprovost/async-reducer#readme",
   "devDependencies": {
-    "babel-eslint": "^8.2.1",
-    "eslint-config-airbnb": "^16.1.0",
+    "babel": "^6.23.0",
+    "babel-cli": "^6.26.0",
+    "babel-core": "^6.26.0",
+    "babel-eslint": "^8.2.2",
+    "babel-plugin-transform-object-rest-spread": "^6.26.0",
+    "babel-preset-env": "^1.6.0",
+    "chai": "^4.1.2",
+    "eslint": "4.18.1",
+    "eslint-config-airbnb-base": "^12.1.0",
+    "eslint-plugin-import": "^2.8.0",
+    "mocha": "^5.0.1",
     "prettier-eslint": "^8.7.4"
+  },
+  "dependencies": {
+    "redux": "^3.7.2"
```

Comment:
shouldn't it be a `peerDependency` instead ?

Reply:
this is a way to specific version of redux, you're forcing everyone to use this one only, and this will probably create nested dependencies (#ugly)